src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "v0.71.1"
+__version__ = "v0.71.2"
```

Comment:
```suggestion
__version__ = "v0.72.0"
```
Bump the minor version since its a new feature